osu.Game/Database/LegacyImportManager.cs

```diff
@@ -98,6 +99,9 @@ public async Task ImportFromStableAsync(StableContent content, bool interactiveL
                 stableStorage = GetCurrentStableStorage();
             }
 
+            if (stableStorage == null)
+                return;
```

Comment:
Free bugfix, courtesy of NRT? Yes please.

Interesting that this never bubbled up on sentry, but on the other side, try fishing that one out of the hundreds of NREs we have there...